src/Compilers/Core/Portable/Symbols/IMethodSymbol.cs

```diff
@@ -263,6 +263,12 @@ public interface IMethodSymbol : ISymbol
         /// </summary>
         IMethodSymbol? PartialImplementationPart { get; }
 
+        /// <summary>
+        /// Return true if this is a partial method definition without a body. If there
+        /// is an implementing body, it can be retrieved with <see cref="PartialImplementationPart"/>.
+        /// </summary>
+        bool IsPartialDefinition { get; }
```

Comment:
>IsPartialDefinition [](start = 13, length = 19)

Should we also add ```PartialImplementation``` API?

Reply:
> Should we also add `PartialImplementation` API?

I did think about doing this, but it didn't seem like it was being requested so I held off. If we see a need for a symmetric query for the other side we can add it at that point IMO.